### ISSUE TITLE
Azure Data Explorer Datasource plugin failed

### ISSUE THREAD

**Issue by dg-eparizzi** [grafana/azure-data-explorer-datasource#259]
**What happened**:
Running Grafana from Docker inside Azure App Service. We were running 7.x but a restart of the service caused to pull down the latest Grafana 8.1.1 and now this data source is all broken.

**What you expected to happen**:
Just to work.

**How to reproduce it (as minimally and precisely as possible)**:
Going to Configuration > Data Sources > Add data source > Azure Data Explorer data source displays the following error

Fetch error: 403 Instantiating https://************.azurewebsites.net/public/plugins/grafana-azure-data-explorer-datasource/module.js Loading plugins/grafana-azure-data-explorer-datasource/module

And I can't add it.

All the dashboards using it are broken.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.1.1
- Plugin version: Not sure. I'm pulling it using the `GF_INSTALL_PLUGINS=grafana-azure-data-explorer-datasource` environment variable on Docker.
- OS Grafana is installed on: Azure App Service Linux

**Comment by chriswue** [comment]
The plugin works fine on my 8.1.3 container instance however one should note that there was a breaking change between Grafana 7.x and 8.x that has lead to the fact that if your plugin path has executable rights set then Grafana will refuse to load the plugin.

In my case I had mapped `/var/lib/grafana/` to an Azure storage account file share which by default has the executable bit set (and it cannot be unset). I ended up just removing the container folder mapping and just installing the plugins in the local container and it works fine

The fact you're getting a 403 indicates that it's most likely this problem

**Comment by joshhunt** [comment]
Hey @dg-eparizzi I know it's been a while, but I took a look into this and couldn't replicate.

Running the docker container locally with `docker run -d -p 3000:3000 -e GF_INSTALL_PLUGINS=grafana-azure-data-explorer-datasource --name grafana grafana/grafana:8.2.0` I was able to connect to our test cluster and run queries as expected.

It sounds like @chriswue has some pointers for how to continue with hosting Grafana on Azure App Service!
